cmd/list/operatorroles/cmd.go

```diff
@@ -173,17 +189,18 @@ func run(cmd *cobra.Command, _ []string) {
 			r.Reporter.Infof(noOperatorRolesPrefixOutput)
 			os.Exit(0)
 		}
-		fmt.Fprintf(writer, "ROLE NAME\tROLE ARN\tVERSION\tMANAGED\n")
+		fmt.Fprintf(writer, "ROLE NAME\tROLE ARN\tCLUSTER ID\tVERSION\tMANAGED\n")
 		for _, operatorRole := range operatorsMap[args.prefix] {
 			awsManaged := "No"
 			if operatorRole.ManagedPolicy {
 				awsManaged = "Yes"
 			}
 			fmt.Fprintf(
 				writer,
-				"%s\t%s\t%s\t%s\n",
+				"%s\t%s\t%s\t%s\t%s\n",
 				operatorRole.RoleName,
 				operatorRole.RoleARN,
+				clusterId,
```

Comment:
This should be operatorRole.ClusterID, right?

Reply:
An empty field was returned when using operatorRole.ClusterID - The above returned the clusterId so maybe I am missing something elsewhere?

Reply:
It is due to this https://github.com/openshift/rosa/pull/1315#discussion_r1236859168

If you put the clusterId variable the user supplied all roles will have the same ID, which wouldn't make a lot of sense